README.md

```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
 
 ![Image showing the parameter file creation options (none, new, and browse)](./images/undo-change-mapping-two.png)
 
+## Linked Template Support
+
+When a [linked template](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/linked-templates#linked-template) is referenced, schema and parameter validations are provided.
+
+Linked template support requires that all top-level parameters have a value. This means that a parameter file is associated with the template OR all top-level parameter definitions have a default value.
+
+Scenarios supported:
+
+- Relative path (requires apiVersion 2020-10-01 or higher of Microsoft.Resources/deployments):
+
+```json
+        {
+            "name": "linkedDeployment1",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    // Relative to current template's folder
+                    "relativePath": "child.json"
+                },
+                "parameters": {
+                }
+            }
+        }
+```
+
+- Full URI:
+
+```json
+        {
+            "name": "linkedDeployment1",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/StephenWeatherford/template-examples/master/linkedTemplates/uri/child.json"
+                },
+                "parameters": {
+                }
+            }
+        }
+```
+
+- Relative to deployed template location
+
+```json
+        {
+            "name": "linkedDeployment1",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    // When the template is deployed, this will be relative to the deployed location
+                    // While editing, this will be relative to the current template's local file folder
+                    "uri": "[uri(deployment().properties.templateLink.uri, 'child.json')]"
+                },
+                "parameters": {
+                }
+            }
+        }
+```
+
+Additional features:
+
+- ["Light-bulb" and snippet support](https://github.com/microsoft/vscode-azurearmtools/blob/mafellin/1225-linked-template-README/README.md#add-missing-parameters) to fill in parameter values for a linked template
+- CTRL-click on relativePath value or click on code lens to navigate to linked template
+
 ## Template navigation
```

Comment:
We should discuss the full validation issue, or point to a wiki page that does.

Reply:
ok, can you assist with a wiki page? I'm not familiar with the history behind it.